db/migrate/20161115153644_add_identity_id_to_epic_queue.rb

```diff
@@ -0,0 +1,5 @@
+class AddIdentityIdToEpicQueue < ActiveRecord::Migration
+  def change
+    add_column :epic_queues, :identity_id, :integer
```

Comment:
Should this be a reference instead? (http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)